### DISCUSSION TITLE
Question about how and why and if...

### DISCUSSION THREAD

**Discussion by jumper-d-1981** [m4dEngi/RemotePlayWhatever#107] [General]
Put into my wishlist in Feb. a game called no man's sky.  At the time it specifically talked about "remote play everywhere" being a part of the game and how that would allow the person who bought it to invite friends to play while the owner played even if they did not have the game.  WELL, it is the summer sale so I thought let's get it while it is 20 bucks cheaper.  never looked at the details.  Looks like at some point since Feb they removed the remote play together option.

Will this work?
I bought it for my son's account and created an alternate account so he could invite me.  BUT when the invite is sent it says the library is locked and he needs to log in on my computer to enable family sharing.  I am so confused.  BUT we were dearly looking forward to a game we could play together.

At some point I was able to start the game on the alt account(have no idea how) and he joined me.  That lasted for 5 or 10 minutes then it kicked me...but at the time we were both our own characters and could see and interact with each other.

Thanks.

BTW, I ran the exe and it says "could not create remote session result 2"  Any thoughts?

**Comment by m4dEngi** [comment]
This application is all about enabling Remote Play Together for any Steam application. Remote Play Together is designed to share screen & control for couch co-op games, enabling local co-op over the network.

Library sharing and Remote play are different beasts. 

So unless you want to play in hotseat mode, this won't work for you.

**Comment by jumper-d-1981** [comment]
Right...I think I am just confused.  Honestly, the only thing I get is a headache when dealing with Steam/Epic etc...

I was just flabbergasted why they would take off the remote play together option...but maybe I am just confused what it all means.  My understanding was that he could play a character and I could play another character and we could each work towards or against each other...like multiplayer in minecraft or whatever.

BUT from what I can tell, he can play and I can watch and sometimes control his guy via a stream...which freaking works really well.  Sorry, first time ever doing games over stream.  

Anyway, unless I am missing something, I think we are stuck.  Buying a second copy of the game...which is a shame but I can deal.  I just miss the old days when I could buy a actual cd and install it and play it and another person could use my disc to multiplay.  They get you coming and going these days.

**Reply by VulcanTourist** [reply]
No Man's Sky doesn't require anything external at all for co-op play: ***it's built into the game***.  You've manufactured a struggle for a game that doesn't have one.
